client/modules/IDE/actions/project.js

```diff
@@ -66,6 +59,30 @@ export function getProject(id, username) {
       });
   };
 }
+export function setIsPrivate(value) {
+  return (dispatch, getState) => {
+    dispatch({
+      type: ActionTypes.SET_IS_PRIVATE_OUTPUT,
+      value
+    });
+    const state = getState();
+    const { id } = state.project;
+    console.log(value);
+    axios.put(`${ROOT_URL}/projects/${id}`, { isPrivate: value }, { withCredentials: true })
+      .then((response) => {
+        if (response.status === 200) {
+          console.log('success');
+        }
+      })
+      .catch((response) => {
+        console.log(response);
```

Comment:
Extra `console.log` calls should be removed such as here, line 74 and line 70 for example.